gbdxtools/images/landsat_image.py

```diff
@@ -48,6 +48,10 @@ def _ndwi_bands(self):
     def _build_graph(cls, _id, band_type="MS", proj=None, **kwargs):
         spec = band_types[band_type]
         landsat = rda.LandsatRead(landsatId=_id, productSpec=spec)
+        if kwargs.get('pansharpen') == True:
```

Comment:
I think this is close but it seems like if `pansharpen=True` we should ignore the use of `band_type` altogether? So no matter what if pansharpen is True, they get a pansharpened image. Make sense?

Reply:
ah yeah I made a similar comment @chelm